src/menu/rom_info.c

```diff
@@ -857,6 +857,8 @@ static rom_err_t save_override (path_t *path, const char *id, int value, int def
 
     if (value == default_value) {
         mini_err = mini_delete_value(rom_info_ini, "custom_boot", id);
+    } else if (strncmp(id, "cheat_codes", strlen("cheat_codes"))) {
+        mini_err = mini_set_bool(rom_info_ini, NULL, id, value);
```

Comment:
something funky going on here... need to check

Reply:
Current thinking is that this block
```
    if (value == default_value) {
        mini_err = mini_delete_value(rom_info_ini, "custom_boot", id);
    } else if (strncmp(id, "cheat_codes", strlen("cheat_codes"))) {
        mini_err = mini_set_bool(rom_info_ini, NULL, id, value);
    } else {
        mini_err = mini_set_int(rom_info_ini, "custom_boot", id, value);
    }
```
is in the wrong place and should be seperate from the ROM info file.

Though I have not looked into where it should actually be.

Reply:
So i wrote this before I had this next idea:
Ideally, the .ini file stores an array or something like that) of cheat titles/indexes (whatever ends up working best), so that rather than having an on-off cheat, the menu will parse previously enabled cheats. 
This approach would ensure errors would ahve been shown to the user at some point, and thus only working/enabled cheats are included